addons/point_of_sale/static/src/app/models/pos_payment.js

```diff
@@ -86,6 +86,10 @@ export class PosPayment extends Base {
         }
         return isPaymentSuccessful;
     }
+
+    updateRefundPaymentLine(refundedPaymentLine) {
+        this.transaction_id = refundedPaymentLine.transaction_id;
```

Comment:
Why is the transaction_id of the refund payment the same as the refunded payment?

Reply:
The `transaction_id` is required for refund requests. Therefore, we first store `refundedPaymentLine.transaction_id` in the payment line. Once the refund response is received, we update it with the new `transaction_id`.